tools/bqtag/bqtag/bqtag.py

```diff
@@ -0,0 +1,800 @@
+"""
+ !/usr/bin/env python
+ -*- coding: utf-8 -*-
+
+ Copyright 2019 Google LLC
```

Comment:
Copyright 2021

Reply:
Changed to :

# Copyright 2021 Google LLC